### PR TITLE
Time to get truly SWOL

### PR DESCRIPTION
**Pull request by Youtubeboy139**
## About The Pull Request

Adds in new things to help players get completely shredded. Shaker Cups got a glow up with a brand new custom sprite that actually has an updating fill overlay now. There are new Protein Packets that can be bought from SweatMAX Vendors so said Shaker Cups have a use beyond fluff containers. The pre-mix Shaker has its own sprite now to differentiate it from the empty cups (same goes for the Protean Mix). Also, please note that the Protein Packets that were added DO NOT have any benefit over the pre-mix drink. They do taste good though.

## How This Contributes To The VOREStation Roleplay Experience

The shaker cups actually have a use beyond fluff now. Honestly don't know why something like this wasn't added in sooner.

## Changelog

:cl:
image: Adds in new custom Protein Shake sprites
image: Adds in new custom Shaker Cup sprites
add: Adds all variations of the Liquid Food packs to the SweatMAX
add: Adds in a Water Bottle to the SweatMAX
add: Adds in new Protein Packets (five flavors in total)
/:cl:

### CODE REVIEW

**Comment by Heroman3003** [comment]
Can you post example screenshots of the new shakers filled with liquid? Sprites look fine, but I just am curious to see how the new fill states look.

**Comment by Youtubeboy139** [comment]
> Can you post example screenshots of the new shakers filled with liquid? Sprites look fine, but I just am curious to see how the new fill states look.

Screenshot 2023-06-21 010938

File: code/modules/food/drinkingglass/shaker.dm [hunk 1]
```diff
@@ -19,19 +19,40 @@
...
+				filling.icon_state = null
+				return
+			if(10 to 19) 	filling.icon_state = "[fillsource]10"
+			if(20 to 29) 	filling.icon_state = "[fillsource]20"
+			if(30 to 39) 	filling.icon_state = "[fillsource]30"
+			if(40 to 49) 	filling.icon_state = "[fillsource]30"
+			if(50 to 59) 	filling.icon_state = "[fillsource]50"
+			if(60 to 69) 	filling.icon_state = "[fillsource]60"
+			if(70 to 79) 	filling.icon_state = "[fillsource]70"
+			if(80 to INFINITY)	filling.icon_state = "[fillsource]80"
```
Comment (Heroman3003) [inline code/modules/food/drinkingglass/shaker.dm:38]: Is there any good reason for this copypaste and not using normal filling_states, just with different numbers?

**Comment by KillianKirilenko** [comment]
> > Can you post example screenshots of the new shakers filled with liquid? Sprites look fine, but I just am curious to see how the new fill states look.
> 
> Screenshot 2023-06-21 010938

Looks much better than the current ones. You can barely tell how full they are at the moment, so this is a significant improvement.

**Pull request merged**